core/sup/src/sup.erl

```diff
@@ -49,19 +50,24 @@ main(CommandLineArgs, Loops) ->
                     'undefined' -> print_invalid_cli_args();
                     F -> list_to_atom(F)
                 end,
+            Arguments = [list_to_binary(Arg) || Arg <- Args],
             Timeout = case props:get_value('timeout', Options) of 0 -> 'infinity'; T -> T * 1000 end,
-            Verbose
+            IsVerbose
                 andalso stdout("Running ~s:~s(~s)", [Module, Function, string:join(Args, ", ")]),
-            case rpc:call(Target, Module, Function, [list_to_binary(Arg) || Arg <- Args], Timeout) of
+
+            case rpc:call(Target, ?MODULE, in_kazoo, [Module, Function, Arguments], Timeout) of
                 {'badrpc', {'EXIT',{'undef', _}}} ->
                     print_invalid_cli_args();
+                {badrpc, {'EXIT', {timeout_value,[{Module,Function,_,_}|_]}}} ->
+                    stderr("Command failed: timeout", []),
```

Comment:
Adding this kinda breaks compat: before on timeouts it would return the stack trace (which only has the given M F As cause that's all the ST there is)